src/coredump/coredump.c

```diff
@@ -813,16 +813,22 @@ static int submit_coredump(
                 return log_error_errno(r, "Failed to drop privileges: %m");
 
         /* Try to get a stack trace if we can */
+        const char *exe = context->meta[META_EXE];
```

Comment:
Use strna here? Things should work without the exe name

Reply:
Hmm, dwfl_core_file_report() seems to be fine will NULL.
I'll rework this.